cfgen/cfgen

```diff
@@ -1431,51 +1500,54 @@ def pool_drives(m0conf: Dict[Oid, Any],
     if ptype is PoolT.dix:
         cas = [(svc_id, m0conf[svc_id].ctrl_id)
                for svc_id in cluster_svcs(m0conf, SvcT.M0_CST_CAS)]
+        path = '/dev/null'
+        nodes = cluster_desc.get('nodes')
+        if nodes:
+            for nodes in nodes:
```

Comment:
`for node is nodes` ? here the variable name and the list name is the same.

Reply:
You also don't need an additional if loop, for loop will terminate is nodes list is empty.

Reply:
Addressed and replaced the variable name.